python/dgl/mock_sparse2/sparse_matrix.py

```diff
@@ -186,28 +186,28 @@ def create_from_coo(
     >>> A = create_from_coo(dst, src)
     >>> print(A)
     SparseMatrix(indices=tensor([[1, 1, 2],
-                                 [2, 4, 3]]),
-                 values=tensor([1., 1., 1.]),
-                 shape=(3, 5), nnz=3)
+            [2, 4, 3]]),
```

Comment:
Can we fix improve the printing of the matrix to have the display on the left?

Reply:
It is controlled by Pytorch. Even a simple 2-D tensor will be printed with indentation.
```
>>> a=torch.tensor([[1,2,3],[4,5,6]])
>>> print(a)
tensor([[1, 2, 3],
        [4, 5, 6]])
```

Reply:
I actually want proper `indentation` when printing a sparse matrix, i.e.

```
SparseMatrix(indices=tensor([[1, 1, 2],
                                                    [2, 4, 3]]),
                        ...)
```

is preferred over

```
SparseMatrix(indices=tensor([[1, 1, 2],
               [2, 4, 3]]),
                        ...)
```

Reply:
Agree that the current printing is not ideal though not the focus of this PR.